addons/sourcemod/scripting/gokz-core/teleports.sp

```diff
@@ -115,7 +115,12 @@ void MakeCheckpoint(int client)
 	{
 		GOKZ_PrintToChat(client, true, "%t", "Make Checkpoint", checkpointCount[client]);
 	}
-	
+
+	if (!GetTimerRunning(client) && AntiCpTriggerIsTouched(client))
```

Comment:
Should this be configurable to be disabled? I guess most people do not use the CP message, but would want to see this. But if you grind a map a lot and know about the areas, this might become quite spammy quickly.

Reply:
Even if you were grinding that map and setting a CP there to practice, you wouldn't spam CPs, so I don't think that's a problem. Can still add that later if it ever becomes annoying.